pkgs/development/tools/rust/cargo-spellcheck/default.nix

```diff
@@ -8,23 +8,25 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-spellcheck";
-  version = "0.8.14";
+  version = "0.9.6";
 
   src = fetchFromGitHub {
     owner = "drahnr";
     repo = pname;
     rev = "v${version}";
-    sha256 = "11r4gzcsbqlflam2rdixc451qw69c46mkf7g0slq6f127is25fgz";
+    sha256 = "0brrpcnsphvl1qn8myrzan0k87ph629gcj6zd54ym9f67qg2nri1";
   };
 
-  cargoSha256 = "1p4iirblk6idvfhn8954v8lbxlzj0gbd8fv4wq03hfrdqisjqcsn";
+  cargoSha256 = "0d0adpgf0j8xvw0dm3gcd6lnksn6p5rz3limc038fg8k84zf7cdb";
 
   buildInputs = lib.optional stdenv.isDarwin Security;
 
   LIBCLANG_PATH = "${libclang.lib}/lib";
 
   checkFlags = [
     "--skip checker::hunspell::tests::hunspell_binding_is_sane"
+    # requires write access to home directory
```

Comment:
This shouldn't be a problem, you could specify `HOME=$(mktemp -d)` in `preCheck`.

Reply:
I did not know that was an option, thank you!  Fixed.